### PR TITLE
4.3 Lessons in Maintance.

### LINKED ISSUE #222: 4.3 - Lessons Maintaince

### PR DESCRIPTION
**Pull request by Hjaltesorgenfrei**

### CODE REVIEW

File: report/43_Lessons-Maintaince.md [hunk 1]
```diff
@@ -1 +1,9 @@
-maintenance
+## Maintenance
+To facilitate maintenance of our the MiniTwit system without we have multiple CI tools which checked that the system was still in working order after changes. These were the Unit Test, API simulator, SonarCloud and Google's Lighthouse.
```
Comment (Jakobis) [inline report/43_Lessons-Maintaince.md]: without?

File: report/43_Lessons-Maintaince.md [hunk 2]
```diff
@@ -1 +1,9 @@
-maintenance
+## Maintenance
+To facilitate maintenance of our the MiniTwit system without we have multiple CI tools which checked that the system was still in working order after changes. These were the Unit Test, API simulator, SonarCloud and Google's Lighthouse. 
+The last of which, Lighthouse, inadvertently showed us we had a issue when merging code to master.
```
Comment (Jakobis) [inline report/43_Lessons-Maintaince.md]: "showed us we had an issue" men næste linje skriver du "not getting spotted". Husk forresten at undgå vi

File: report/43_Lessons-Maintaince.md [hunk 3]
```diff
@@ -1 +1,9 @@
-maintenance
+## Maintenance
+To facilitate maintenance of our the MiniTwit system without we have multiple CI tools which checked that the system was still in working order after changes. These were the Unit Test, API simulator, SonarCloud and Google's Lighthouse. 
+The last of which, Lighthouse, inadvertently showed us we had a issue when merging code to master. 
+But due to the issue not getting spotted and misconfiguration of which CI checks had to pass before code could be merged, we ended up breaking the register page on our website. This was first realized when a user reported the error to us. 
+If one were to look at the [pull request (Fix SonarCloud issues #186)](https://github.com/jlndk/devoops/pull/186) it is quite clear that a issue came up when Lighthouse tried to reach the register page, as all the check came back red with a score of 0. 
+
+
+There were similar issues which our CI tools caught but that were missed. These issues that got through to Master and got deployed could probably have been caught in the pull request, if a more rigorous and defined review process was defined.
```
Comment (Jakobis) [inline report/43_Lessons-Maintaince.md]: Undgå også our

**Review by Jakobis — CHANGES_REQUESTED** [review]

**Review by Jakobis — APPROVED** [review]

**Review by Duckapple — CHANGES_REQUESTED** [review]
Line 7 feels like you are repeating yourself in 2 sentences. 
Line 5 has "...clear that a issue...", which should be "an".

**Review by Duckapple — APPROVED** [review]

**Pull request merged**